bundles/automation/org.eclipse.smarthome.automation.api/src/main/java/org/eclipse/smarthome/automation/handler/ActionHandler.java

```diff
@@ -30,6 +30,6 @@
      *            in form: ModuleId.outputId
      * @return values map of values which must be set to outputs of the {@link Action} (may be null).
      */
-    public Map<String, Object> execute(Map<String, ?> context);
+    public Map<String, Object> execute(Map<String, Object> context);
```

Comment:
Shouldn't we change the JavaDoc to inform the implementer that the context argument is unmodifiable?